etna/lib/etna/clients/polyphemus/client.rb

```diff
@@ -26,6 +26,22 @@ def configuration(configuration_request = ConfigurationRequest.new)
 
         ConfigurationResponse.new(json)
       end
+
+      def job(job_request = JobRequest.new, &block)
+        # Because this is a streaming response, just yield the response back.
+        #   The consumer will have to iterate over the response.read_body, like
+        #
+        # polyphemus_client.job(request) do |response|
+        #   response.read_body do |fragment|
+        #     <fragment contains a chunk of text streamed back from the server>
+        #   end
+        # end
+        @etna_client.post(
+          "/#{job_request.project_name}/job",
+          job_request) do |res|
+            yield res
```

Comment:
In this case the yield is redundant with

```
        @etna_client.post(
          "/#{job_request.project_name}/job",
          job_request, &block)
```

But not worth waiting on a whole set of tests to change :P

Reply:
I didn't realize that, thanks for pointing that out!